cluster/gce/gci/configure.sh

```diff
@@ -255,6 +255,40 @@ function download-network-template {
   )
 }
 
+function download-vpc-template {
```

Comment:
download file seems a very common op; suggest to refactor and reuse.

Reply:
Consider do this and should be quick fix.

Reply:
Use new function 'download-template-file' to replace 3 functions - download-network-template, download-vpc-template, download-subnet-template